pkgs/applications/window-managers/i3/balance-workspace.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, i3ipc }:
+
+buildPythonPackage rec {
+  pname = "i3-balance-workspace";
+  version = "1.8.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1gndzrwff8gfdqjjxv4zf2h2k0x7y97w1c3mrjpihz8xd0hbnk4d";
+  };
+
+  propagatedBuildInputs = [ i3ipc ];
+  doCheck = false; # no test
```

Comment:
Can this package be imported? If so please add a ``pythonImportsCheck = [ "i3-balance-workspace" ];``.

Reply:
i3-balance-workspace is a standalone executable program which doesn't seem to be meant to be used as a Python library.

Reply:
I just tested and it indeed works :+1: